packages/slice-machine/lib/models/common/widgets/sliceBuilderArray.ts

```diff
@@ -4,6 +4,7 @@ export default [
   Widgets.StructuredText,
   Widgets.Image,
   Widgets.Link,
+  Widgets.LinkToMedia,
```

Comment:
there are way too many file exporting the widgets 😂

Reply:
Terrible 📦